Gems/GradientSignal/Code/Source/GradientTransform.cpp

```diff
@@ -30,31 +29,17 @@ namespace GradientSignal
             m_inverseTransform.SetRow(2, AZ::Vector4::CreateZero());
         }
 
-        // Set up the appropriate wrapping transform function for the the given wrapping type.
-        // Also note that ClampToZero is the only wrapping type that allows us to return a "pointIsRejected" result
-        // for points that fall outside the shape bounds.
-        if (m_shapeBounds.IsValid())
+        // If we have an invalid shape bounds, reset the wrapping type back to None.  Wrapping won't work without a valid bounds.
```

Comment:
Isn't "bounds" plural?

```
// If we have invalid shape bounds, reset the wrapping type back to None. Wrapping won't work without valid bounds.
```

Reply:
Sort of? The bounds in this case is/are represented through a singular AABB, in a single variable, which is what I was implicitly referring to. But I see your point, I'll change it.